app.js

```diff
@@ -97,6 +100,18 @@ const runDaemon = async () => {
                     },
                 });
             }
+
+            // "random problem" guild command
+            if (name === "randomproblem") {
+                const message = await leetcode.getRandomProblemMessage();
+                return res.send({
+                    type: InteractionResponseType.CHANNEL_MESSAGE_WITH_SOURCE,
+                    data: message,
+                });
+            }
+
+            // TODO: "problem from set" guild command
```

Comment:
you could maybe link this to a ticket, TODO(#randomnumber): Add problem set guild command

Reply:
Done, created #80